docs/tutorials/10_pvqd.ipynb

```diff
@@ -0,0 +1,563 @@
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit.circuit import QuantumCircuit, ParameterVector\n",
+    "\n",
+    "theta = ParameterVector(\"th\", 5)\n",
+    "ansatz = QuantumCircuit(2)\n",
+    "ansatz.rx(theta[0], 0)\n",
+    "ansatz.rx(theta[1], 1)\n",
+    "ansatz.rzz(theta[2], 0, 1)\n",
+    "ansatz.rx(theta[3], 0)\n",
+    "ansatz.rx(theta[4], 1)\n",
+    "\n",
+    "# you can try different circuits, like:\n",
+    "# from qiskit.circuit.library import EfficientSU2\n",
+    "# ansatz = EfficientSU2(2, reps=1)\n",
+    "\n",
+    "ansatz.draw(\"mpl\", style=\"iqx\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With this ansatz, the $|00\\rangle$ state is prepared if all parameters are 0. Hence we'll set the initial parameters to $\\theta_0 = 0$:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "\n",
+    "initial_parameters = np.zeros(ansatz.num_parameters)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Before running the p-VQD algorithm, we need to select the backend and how we want to calculate the expectation values. Here, we'll perform exact statevector simulations (which are still very fast, as we investigate a 2 qubit system) through the reference primitive implementations found in `qiskit.primitives`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit.primitives import Sampler, Estimator\n",
+    "from qiskit_algorithms.state_fidelities import ComputeUncompute\n",
+    "\n",
+    "# the fidelity is used to evaluate the objective: the overlap of the variational form and the trotter step\n",
+    "sampler = Sampler()\n",
+    "fidelity = ComputeUncompute(sampler)\n",
+    "\n",
+    "# the estimator is used to evaluate the observables\n",
+    "estimator = Estimator()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since p-VQD performs a classical optimization in each timestep to determine the best parameters for the projection, we also have to specify the classical optimizer. As a first example we're using BFGS, which typically works well in statevector simulations, but later we can switch to gradient descent."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit_algorithms.optimizers import L_BFGS_B\n",
+    "\n",
+    "bfgs = L_BFGS_B()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can define p-VQD and execute it!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit_algorithms.time_evolvers.pvqd import PVQD\n",
```

Comment:
```suggestion
    "from qiskit_algorithms import PVQD\n",
```
Lets used the same place we document it etc to import it. This even went as far as to import from the pvqd.py module directly, deeper even that from time_evolvers